src/main/java/racingcar/vo/CarList.java

```diff
@@ -0,0 +1,68 @@
+package racingcar.vo;
+
+import java.util.Collections;
+import java.util.List;
+import java.util.stream.Collectors;
+import racingcar.model.Car;
+
+public class CarList {
+    private static final String STRING_WINNER_DELIMITER = ", ";
+    private final List<Car> carList;
+
+    public CarList(final List<Car> carList) {
+        this.carList = carList;
+    }
+
+    public List<Car> getCarList() {
+        return carList.stream()
+                .collect(Collectors.toUnmodifiableList());
+    }
+
+    public void moveForward() {
+        carList.stream()
+                .forEach(Car::moveForward);
```

Comment:
`forEach` 는 `stream()` 을 쓰지 않아도 사용할 수 있다.

```suggestion
        carList.forEach(Car::moveForward);
```